CHANGELOG.md

```diff
@@ -102,6 +102,11 @@ Notes: web developers are advised to use [`~` (tilde range)](https://github.com/
    - Attaching files will no longer remove previously attached files
    - Updated Fluent theme to use the new attachment preview feature
 - Added collapsible activity and activity with abstract handling, in PR [#5506](https://github.com/microsoft/BotFramework-WebChat/pull/5506), in PR [#5513](https://github.com/microsoft/BotFramework-WebChat/pull/5513), by [@OEvgeny](https://github.com/OEvgeny)
+- Added `disableFileUpload` flag to completelly disable file upload feature,
+- Deprecated `hideUploadButton` in favor of `disableFileUpload`. Added rectification logic in `patchStyleOptionsFromDeprecatedProps.js`.
+- Updated `BasicSendBoxToolbar` to rely solely on `disableFileUpload`.
+- Improved test to simulate keyboard and mouse behavior instead of DOM-only checks.
+  in PR [#5508](https://github.com/microsoft/BotFramework-WebChat/pull/5508), by [@JamesNewbyAtMicrosoft](https://github.com/JamesNewbyAtMicrosoft)
```

Comment:
```suggestion
- Added `disableFileUpload` flag to completelly disable file upload feature, in PR [#5508](https://github.com/microsoft/BotFramework-WebChat/pull/5508), by [@JamesNewbyAtMicrosoft](https://github.com/JamesNewbyAtMicrosoft)
  - Deprecated `hideUploadButton` in favor of `disableFileUpload`. Added rectification logic in `patchStyleOptionsFromDeprecatedProps.js`.
  - Updated `BasicSendBoxToolbar` to rely solely on `disableFileUpload`.
  - Improved test to simulate keyboard and mouse behavior instead of DOM-only checks.

```